src/python/benchUtil.py

```diff
@@ -1498,6 +1498,7 @@ def getAntClassPath(checkout):
   cp.append(core_jar_file)
   cp.append('%s/lucene/build/core/classes/test' % path)
   cp.append('%s/lucene/build/sandbox/classes/java' % path)
+  cp.append('%s/lucene/build/test-framework/classes/test' % path)
```

Comment:
I had to add this dependency or else I got class loader errors 🤔 

```
Exception in thread "main" java.util.ServiceConfigurationError: org.apache.lucene.codecs.Codec: Provider org.apache.lucene.codecs.TestMinimalCodec$MinimalCodec could not be instantiated
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:586)
	at java.base/java.util.ServiceLoader$ProviderImpl.newInstance(ServiceLoader.java:813)
...
Caused by: java.lang.NoClassDefFoundError: org/apache/lucene/tests/util/TestUtil
	at org.apache.lucene.codecs.TestMinimalCodec$MinimalCodec.<init>(TestMinimalCodec.java:83)
	at org.apache.lucene.codecs.TestMinimalCodec$MinimalCodec.<init>(TestMinimalCodec.java:86)
```

Reply:
I believe you got this because `core/classes/test` is on the classpath, but I don't know why we need it and it feels wrong to me that luceneutil needs test classes. Maybe we should remove it instead (line 1499 in the same file)?

Reply:
Removing `core/classes/test` seemed to work fine, at least with gradle (I ran through a couple full tests).